questions/npc.md

```diff
@@ -0,0 +1,60 @@
+# How to create / customize an NPC?
+This article will explain how to create NPCs and add commands to it.
+
+> [!NOTE]
+> This is for the entity in Bedrock edition called [`NPC`](https://minecraft.wiki/w/NPC)
+
+## Basics of an NPC
+
+For spawning an NPC, you will need to use the `/summmon` or `/give` command:
+
+    /summon npc
+    /give @s spawn_egg 1 51
+
+For removing the NPC just left click it while you are in creative, an NPC is invulnerable if you try to kill it with other methods (unless when falling into the void), but it can be moved with water or pistons.
```

Comment:
wouldn't a reference to the kill command be in order here? instead of just punching it to death? 😄

Reply:
It’s not like an attack, I tested and if you right click it it will de spawn, no death animation. (Only works in creative) if you want a video of how it “despawns” I could probably send it to you tomorrow